warc_gpt/static/index.js

```diff
@@ -39,7 +39,7 @@ const askButton = document.querySelector("#ask");
  * @returns {string}
  */
 const sanitizeString = (string, convertLineBreaks = true) => {
-  string = string.trim()
+  string = string
```

Comment:
trim() needed to be removed here to preserve space between words as they come in